src/App.tsx

```diff
@@ -15,48 +16,111 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+enum SortType {
+  NAME = 'name',
+  LENGTH = 'length',
+  NONE = 'none',
+}
+
+function sortArray(
+  goods: string[],
+  howSort: SortType,
```

Comment:
`howSort` it's not a correct name, because it's sounds as question with some actions
it's better to rename it to `type` or `sortType`